src/modules/site-v2/templates/tools/pairwise_indel_finder/submit.html

```diff
@@ -183,7 +167,7 @@ <h1 class="modal-title fs-5" id="generate-primers-modal-label">Thank You</h1>
 		}
 	});
 
-  fetch_strains().then((data) => {
+  fetch_json("{{ url_for('pairwise_indel_finder.get_strains') }}").then((data) => {
```

Comment:
Add a `.catch(...)` promise chain to handle possible errors (HTTP network errors, .json parsing errors, etc).

Reply:
Sort of fixed in d439320a0fdf14e797c555f9b5cd70a959261c31 -- see discussion at https://github.com/AndersenLab/CAENDR/pull/166#discussion_r1154930052.